Generates this error:

```
/tensorflow/compiler/xla/service/gpu/nvptx_compiler.cc:23:10: error: module //tensorflow/compiler/xla/service/gpu:nvptx_compiler_impl does not depend on a module exporting 'third_party/llvm/llvm-project/llvm/include/llvm/IRReader/IRReader.h'
see http://go/cpp-features#layering_check; to fix run:
build_cleaner //third_party/tensorflow/compiler/xla/service/gpu:nvptx_compiler_impl
```

--- a/tensorflow/compiler/xla/service/gpu/BUILD
+++ b/tensorflow/compiler/xla/service/gpu/BUILD
@@ -1529,6 +1529,8 @@ cc_library(
         "//tensorflow/stream_executor:stream_executor_headers",
         "//tensorflow/stream_executor/cuda:cuda_diagnostics",
         "//tensorflow/stream_executor/gpu:asm_compiler",
+        "@llvm-project//llvm:IRReader",
+        "@llvm-project//llvm:Support",
     ]) + ["//tensorflow/stream_executor/gpu:gpu_driver_header"],
 )
 

